utils/cluster/Node.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiPromise } from "@polkadot/api";
+import { initApi } from "../../utils/api";
+import { testLog } from "../Logger";
+
+export { Node };
+
+class Node {
+  name: string;
+  connected: boolean;
+  api?: ApiPromise;
+
+  firstBlock?: number;
+  lastBlock?: number;
+  lastHash?: string;
+  hashes: Set<string> = new Set();
+  blockHashes: Map<number, string> = new Map();
+  subscription: any;
+
+  constructor(name: string) {
+    this.name = name;
+    this.connected = false;
+  }
+
+  async connect(wsPath: string): Promise<void> {
+    if (!this.connected) {
+      this.api = await initApi(wsPath);
+    }
+  }
+
+  async start(): Promise<void> {
```

Comment:
totally optional :)

It seems we are not starting the node, but the "hashSubscriber" , maybe rename to 
SubscribeToHash instead of start ?